### ISSUE TITLE
Invite players to join the host in a multiplayer lobby

### ISSUE THREAD

**Issue by samcf** [samcf/ogre.tools#24]
Sharing the player view window over Discord or Zoom is a decent, effective way to create a collaborative experience. There are some pretty serious limitations, however.

- The operating system doesn't want to update windows that are hidden behind other windows; this forces the host to keep the player view window visible on their own computer.
- Players are limited to telling the host what they'd like their character to do and cannot move their own tokens.
- It is much less of an immersive experience; far removed from the traditional round table setup. Players feel more immersed in the experience if they can directly control the objects on the canvas.
- The visual quality and detail is vastly improved by rendering directly on the player's browser instead of through a video conference.

Many other virtual tabletops allow hosts to invite players to participate in a multiplayer lobby through the browser. This story represents the first step towards a more collaborative tabletop experience.

- The host may turn their session into a public lobby which can be accessed by anyone with the private URL that is shared by the host to the players.
- The players navigate to the given URL and can see the canvas and all the publicly visible elements on it.
- Players are restricted to only being able to move the camera around.

**Comment by samcf** [comment]
Example WebSocket message (EDN)

```edn
{:type   :message
 :params {:name :initial-connection}
 :sender "1e51189b-e3ea-4e0d-b669-d7c8c1f8b637"}
```

**Comment by samcf** [comment]
Server message sent to client upon connecting

```edn
{:type   :event
 :name   :connection-received
 :params {:uuid "1e51189b-e3ea-4e0d-b669-d7c8c1f8b637"}}
```

**Comment by samcf** [comment]
DataScript transaction example

```edn
{:type   :transaction
 :name   :camera/translate
 :args   [230 400]
 :sender "3e584c31-41c3-4dbd-8498-e5ffc5d94c1a"}
```

**Comment by samcf** [comment]
Resolved in #28
